src/controllers/dynakube/version/oneagent.go

```diff
@@ -55,6 +55,10 @@ func (updater oneAgentUpdater) IsPublicRegistryEnabled() bool {
 	return updater.dynakube.FeaturePublicRegistry()
 }
 
+func (updater oneAgentUpdater) IsClassicFullStackEnabled() bool {
```

Comment:
this only needs to be checked in the oneagent updater and could be done in its `IsPublicRegistryEnabled`

like: 
```
func (updater oneAgentUpdater) IsPublicRegistryEnabled() bool {
	return updater.dynakube.FeaturePublicRegistry() && ! updater.dynakube.ClassicFullStackMode()
}
```